tests/migration/Core/V6_7/Migration1739197440ChangeCmsBlockDefaultMarginTest.php

```diff
@@ -0,0 +1,106 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_7;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Exception;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\DatabaseTransactionBehaviour;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\Migration\V6_7\Migration1739197440ChangeCmsBlockDefaultMargin;
+
+/**
+ * @internal
+ */
+#[CoversClass(Migration1739197440ChangeCmsBlockDefaultMargin::class)]
+class Migration1739197440ChangeCmsBlockDefaultMarginTest extends TestCase
+{
+    use DatabaseTransactionBehaviour;
+    use KernelTestBehaviour;
+
+    private Connection $connection;
+
+    protected function setUp(): void
+    {
+        $this->connection = $this->getContainer()->get(Connection::class);
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function testChangeCmsBlockDefaultMargin(): void
+    {
+        $migration = new Migration1739197440ChangeCmsBlockDefaultMargin();
+
+        $migration->update($this->connection);
```

Comment:
please execute the update method twice, so we can make sure, the migration can be executed multiple times.

Reply:
Fixed!